Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Volunteer.hs

```diff
@@ -80,6 +82,9 @@ assignCreateAndStartOtpRide _ Common.AssignCreateAndStartOtpRideAPIReq {..} = do
 
   ride <- DRide.otpRideCreate requestor rideOtp booking
   let driverReq = RideStart.DriverStartRideReq {rideOtp, point, requestor}
+  fork "sending dashboard sms - start ride" $ do
+    mride <- runInReplica $ QRide.findById ride.id >>= fromMaybeM (RideDoesNotExist ride.id.getId)
```

Comment:
Can't you use the ride already being fetched at line#83?

Reply:
The previous ride is of type = DriverRideRes
I need to use type = Ride 
That's why fetched the Ride